idd/Energy+.idd.in

```diff
@@ -58691,6 +58696,7 @@ Coil:Heating:WaterToAirHeatPump:VariableSpeedEquationFit,
         \reference HeatingCoilsWaterToAirVSHP
         \reference-class-name validBranchEquipmentTypes
         \reference validBranchEquipmentNames
+        \reference AFNCoilNames
```

Comment:
These seem...fine.  As long as they are supported coils for the AFN now, which I suppose they are after looking at the Solver code.